.github/workflows/build-and-sign-image.yml

```diff
@@ -3,22 +3,18 @@
 name: Build and sign image
 
 on:
-  pull_request:
-    types:
-      - closed
-    branches:
-      - "main"
-
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Is the `tags` directive compatible with `branches`?  Might be good to enforce the branch too.  Otherwise we could build a release of a branch if a tag is created.

Reply:
Yeah, this is an issue. For the time being let's just be diligent about this (note: I used this _feature_ while working on this, the ability to test Actions is so limited I needed to be able to trigger them from a branch to test).  

ChatGPT interaction:

> If you create a tag on a feature branch and push it, the workflow will trigger if the tag matches the specified pattern (v* in the example). The workflow is not conditioned by the branch but by the tag's presence and pattern.
> 
> To illustrate, let's say you have a feature branch named feature/awesome-feature and you create a tag named v1.2.3 on this branch. When you push this tag to GitHub, the above workflow will be triggered because it meets the tag pattern criteria (v*), even though the tag was created on a branch that isn't explicitly listed in the workflow file.
> 
> If you need to have specific actions for tags created from specific branches, you would typically handle that logic within the steps of your workflow, using conditional statements and possibly scripts to determine the origin branch of the tag. However, this can be complex and is not natively supported as a straightforward configuration in GitHub Actions.
>